tests/cms/views/utils.py

```diff
@@ -46,7 +49,39 @@ def check_view_status_code(
         # If the post data is a string, assume json as content type
         if isinstance(post_data, str):
             post_kwargs["content_type"] = "application/json"
-        response = client.post(url, **post_kwargs)
+        if view_name == "mediacenter_replace_file":
```

Comment:
I'm not insisting here, but would like to share some thoughts to consider.
I see some advantages in implementing tests for views as separate functions (rather than all in this one):
- if-s by specific view names increase complexity
- tests could be run separately per suite (as I understand, I can't run media library-only tests now?)
- we can easily enrich the tests with view-specific checks, e.g., in a database (to verify that media files have actually been created/updated/deleted)
- tests will be more like user scenarios

I'm open to other opinions

Reply:
Thank you for the advice 🙂 I find it a good idea too and separated the tests for media library.